credentials/alts/alts_test.go

```diff
@@ -22,12 +22,21 @@
 package alts
 
 import (
+	"context"
+	"net"
 	"reflect"
+	"runtime"
+	"sync"
 	"testing"
+	"time"
 
 	"github.com/golang/protobuf/proto"
+	grpc "google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/alts/internal/handshaker/service"
 	altspb "google.golang.org/grpc/credentials/alts/internal/proto/grpc_gcp"
+	"google.golang.org/grpc/credentials/alts/internal/testutil"
 	"google.golang.org/grpc/internal/grpctest"
+	testpb "google.golang.org/grpc/interop/grpc_testing"
```

Comment:
Please split this into two imports:

```
	testpb "google.golang.org/grpc/interop/grpc_testing"
	testgrpc "google.golang.org/grpc/interop/grpc_testing"
```

In OSS, generated code for messages are services are placed in the same package. But this is not the case in google3. So, unless we have separate imports for the two, there is no way we can rewrite them at import time to point to the appropriate ones in google3.

Reply:
Done.